docs/UserGuide.md

```diff
@@ -123,17 +123,29 @@ Examples:
     email: [johnd@example.com](mailto:johnd@example.com)  
     address: John street, block 123, #01-011
 
-### Locating persons by name: `find`
+### Locating clients by keywords: `find`
 
-This command allows you to find a specific client in your client list by their **name**
+This command allows you to find a specific client in your client list by their **name**, and optionally by other fields too
 
-Format: `find n/NAME`
+Format: `find n/NAME [a/ADDRESSKEYWORD] [e/EMAILKEYWORD] [t/TAGKEYWORD] [k/KINKEYWORD] [d/DESCKEYWORD] [p/PHONEKEYWORD]`
```

Comment:
Not sure if there are parameter constraints, but if there are you should put it here